docs/operating-scylla/nodetool-commands/tasks-set-ttl.rst

```diff
@@ -0,0 +1,18 @@
+Nodetool tasks-set-ttl
+======================
+**tasks-set-ttl** - Sets task_ttl value in seconds, i.e. time for which tasks are kept in task manager after
+they are finished. Returns previous ttl value.
+
```

Comment:
Better to have separate ttl and set-ttl subcommands. It's not customary to have exchange style commands in CLI.